package.json

```
@@ -26,6 +26,7 @@
     "static:serve": "yarn static:build && npx servor out app.html 5000",
     "theme:generate-typing": "chakra-cli tokens ./src/theme/theme.ts",
     "theme:generate-icons": "svgr --config-file src/components/Icons/svgr.config.js src/components/Icons/svg-sources",
+    "new:story": "plop story",
```

I should have done something like this:
```suggestion
    "generate": "plop",
```

Here we create an alias of plop, so we can do the following commands without to define new scripts in package.json:

```bash
yarn generate component
yarn generate page
```

We could have use plop, but I personnaly don't like `yarn plop component` because it doesn't mean anything ^^